client/src/components/menubar/index.js

```diff
@@ -42,7 +44,8 @@ import * as globals from "../../globals";
   redoDisabled: state["@@undoable/future"].length === 0,
   aboutLink: state.config?.links?.["about-dataset"],
   disableDiffexp: state.config?.parameters?.["disable-diffexp"] ?? false,
-  diffexpMayBeSlow: state.config?.parameters?.["diffexp-may-be-slow"] ?? false
+  diffexpMayBeSlow: state.config?.parameters?.["diffexp-may-be-slow"] ?? false,
+  centroidToggle: state.centroidLabels.toggle
```

Comment:
very vague name - suggest something more descriptive, eg, `displayCentroidLabels` or the like....

Reply:
renamed to `showCentroidLabels`